packages/google_maps_flutter/google_maps_flutter/example/lib/animate_camera.dart

```diff
@@ -156,14 +180,33 @@ class AnimateCameraState extends State<AnimateCamera> {
                   onPressed: () {
                     mapController?.animateCamera(
                       CameraUpdate.zoomTo(16.0),
+                      configuration: _cameraUpdateAnimationConfiguration,
                     );
                   },
                   child: const Text('zoomTo'),
                 ),
               ],
             ),
           ],
-        )
+        ),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            const Text(
+              'With configuration\n(10 second duration)',
+              textAlign: TextAlign.right,
+            ),
+            const SizedBox(width: 5),
+            Switch(
+              value: _cameraUpdateAnimationConfiguration != null,
```

Comment:
These should all be "Duration" rather than "Configuration" once the other change is made.